cc_plugin_glider/glider_dac.py

```diff
@@ -41,6 +45,62 @@ class GliderCheck(BaseNCCheck):
         'Slocum Glider'
     }
 
+    def __init__(self):
+        # try to get the sea names table
+        ncei_base_table_url = 'https://gliders.ioos.us/ncei_authority_tables/'
+        # might refactor if the authority tables names change
+        table_type = {'project': 'projects.txt',
+                      'platform': 'platforms.txt',
+                      'instrument': 'instruments.txt',
+                      'institution': 'institutions.txt'}
+        # some top level attrs map to other things
+        var_remap = {'platform': 'id',
+                     'instrument': 'make_model'}
+
+        self.auth_tables = {}
+        for global_att_name, table_file in six.iteritems(table_type):
+            # instruments have to be handled specially since they aren't
+            # global attributes
+            table_loc = urljoin(ncei_base_table_url, table_file)
+            self.auth_tables[global_att_name] = GliderCheck.request_resource(table_loc, None,
+                                                    lambda s: set(s.splitlines()))
+
+        # handle NCEI sea names table
+        sea_names_url = 'https://www.nodc.noaa.gov/General/NODC-Archive/seanames.xml'
+        def sea_name_parse(text):
+            """Helper function to handle utf-8 parsing of sea name XML table"""
+            utf8_parser = etree.XMLParser(encoding='utf-8')
+            tree = etree.fromstring(text.encode('utf-8'), parser=utf8_parser)
+            return set(tree.xpath('./seaname/seaname/text()'))
+
+        self.auth_tables['sea_name'] = GliderCheck.request_resource(sea_names_url,
+                                                                    None,
+                                                                    sea_name_parse)
+
+    @classmethod
+    def request_resource(cls, url, backup_resource, fn):
```

Comment:
Setting `backup_resource` (cached file resource) isn't currently reachable in the code, which is one reason why this is a draft PR.  We could potentially use the model the CF checker uses by utilizing environment variables.